src/iop/basecurve.c

```diff
@@ -178,65 +180,61 @@ typedef struct basecurve_preset_t
+    // contributed by chrik5
+    { "Pentax K-5",           "Pentax",            "Pentax K-5",           0, FLT_MAX, {{{{ 0.000000, 0.000000 }, { 0.004754, 0.002208 }, { 0.009529, 0.004214 }, { 0.023713, 0.013508 }, { 0.031866, 0.020352 }, { 0.046734, 0.034063 }, { 0.059989, 0.052413 }, { 0.088415, 0.096030 }, { 0.136610, 0.190629 }, { 0.174480, 0.256484 }, { 0.205192, 0.307430 }, { 0.228896, 0.348447 }, { 0.286411, 0.428680 }, { 0.355314, 0.513527 }, { 0.440014, 0.607651 }, { 0.567096, 0.732791 }, { 0.620597, 0.775968 }, { 0.760355, 0.881828 }, { 0.875139, 0.960682 }, { 1.000000, 1.000000 }}}, { 20 }, { MONOTONE_HERMITE }}, 0, 1 },
+    // contributed by Togan Muftuoglu - ed: slope is too aggressive on shadows
+    //{ "Nikon D90", "NIKON", "D90", 0, FLT_MAX, { { { { 0.000000, 0.000000 }, { 0.015520, 0.012248 }, { 0.097950, 0.251013 }, { 0.301515, 0.621951 }, { 0.415513, 0.771384 }, { 0.547326, 0.843079 }, { 0.819769, 0.956678 }, { 1.000000, 1.000000 } } }, { 8 }, { MONOTONE_HERMITE } }, 0, 1 },
+    // contributed by Edouard Gomez
+    { "Nikon D90",            "NIKON CORPORATION", "NIKON D90",            0, FLT_MAX, {{{{ 0.000000, 0.000000 }, { 0.011702, 0.012659 }, { 0.122918, 0.289973 }, { 0.153642, 0.342731 }, { 0.246855, 0.510114 }, { 0.448958, 0.733820 }, { 0.666759, 0.894290 }, { 1.000000, 1.000000 }}},                                                                                                                                                                                                                                                                                                 { 8 },  { MONOTONE_HERMITE }}, 0, 1 },
+    // contributed by Pascal Obry
+    { "Nikon D800",           "NIKON",             "D800",                 0, FLT_MAX, {{{{ 0.000000, 0.000000 }, { 0.001773, 0.001936 }, { 0.009671, 0.009693 }, { 0.016754, 0.020617 }, { 0.024884, 0.037309 }, { 0.048174, 0.107768 }, { 0.056932, 0.139532 }, { 0.085504, 0.233303 }, { 0.130378, 0.349747 }, { 0.155476, 0.405445 }, { 0.175245, 0.445918 }, { 0.217657, 0.516873 }, { 0.308475, 0.668608 }, { 0.375381, 0.754058 }, { 0.459858, 0.839909 }, { 0.509567, 0.881543 }, { 0.654394, 0.960877 }, { 0.783380, 0.999161 }, { 0.859310, 1.000000 }, { 1.000000, 1.000000 }}}, { 20 }, { MONOTONE_HERMITE }}, 0, 1 },
   // clang-format on
 };
 static const int basecurve_camera_presets_cnt = sizeof(basecurve_camera_presets) / sizeof(basecurve_preset_t);
 
 static const basecurve_preset_t basecurve_presets[] = {
   // clang-format off
-  // smoother cubic spline curve
```

Comment:
likewise, just reformatting. Again I'm not against it, but it makes the actual patch more difficult to read.